tests/framework/clients/bucket.go

```diff
@@ -157,9 +157,9 @@ func (b *BucketOperation) CheckBucketNotificationSetonRGW(namespace, storeName,
 	s3AccessKey, _ := helper.BucketClient.GetAccessKey(obcName)
 	s3SecretKey, _ := helper.BucketClient.GetSecretKey(obcName)
 	if tlsEnabled {
-		s3client, err = rgw.NewInsecureS3Agent(s3AccessKey, s3SecretKey, s3endpoint, "", true)
+		s3client, err = rgw.NewInsecureS3Agent(s3AccessKey, s3SecretKey, s3endpoint, storeName, true)
```

Comment:
If a different region is specified in the storage class, will this client connection work with the store name? Or does it need to match the storage class?

Reply:
> `us-east-1` is still used in the integration tests in ceph_base_object_test.go. How about changing that to something else so we can see the non-default works?
@travisn : do u want to check whether with regions and see s3 client works or not right? It more or like s3 client kind of test, but I can definitely add those

Reply:
@travisn : I wondered magic behind `us-east-1` as region value. The actual value for s3 client needs to `storeName` and ideally we should have filled same value for storage class. The `us-east-1` works for RGW, I am guessing it skips checks(or maybe bug)

Reply:
@travisn : The only `create bucket` request needs the correct region value, all the objects operation works fine with the 
invalid region for s3 client. So I guess I need to add a test case for `cephobjectstoreuser` with creating a bucket to test this out

[Another update]: `us-east-1` as a region no longer valid in master, it works with pacific though. So IMO we might seen similar issue when we start using new ceph versions

Reply:
So if we remove region from the default examples in rook, it will work with ceph pacific and master? Or will we need different examples for pacific vs quincy? And to be clear, does the ceph master change also affect quincy, or just post-quincy?

Reply:
Yeah, it will work, this fills internal by Rook. Earlier we are using `us-east-1` from this patch we add the actual zone of the rgw server that's the difference.
The `us-east-1` worked in quincy as well, but I don't see much code difference in that path for master and Quincy. Need to have a bit of time to understand that

Reply:
I need to take back about `us-east-1` region value, this will work with the master and Quincy branch. Earlier it was a typo mistake from my end

Reply:
The documented examples we are working on in the other comments only apply to Quincy and newer, correct? So what is the behavior for region in Pacific? The docs need to give that detail, especially since Pacific is still the default version with Rook.

Reply:
I'm still confused about how users can understand the region. If `us-east-1` is a workaround and a hack, maybe we should just keep that for the simple, local rgw case. What does the user gain by using a different region? Seems like it should only be important for external or multi-site where we need the region name in order to connect. But if it's just the local cluster, can we hide the region from the user so they don't need to worry about it?

Reply:
Let's find a time to discuss this next week since I'm having such a hard time. :)

Reply:
@travisn: Sure :). Sounds good to me